x-pack/plugins/apm/server/lib/traces/get_trace.ts

```diff
@@ -10,7 +10,27 @@ import { Span } from '../../../typings/es_schemas/Span';
 import { Transaction } from '../../../typings/es_schemas/Transaction';
 import { Setup } from '../helpers/setup_request';
 
-export type TraceAPIResponse = Array<Transaction | Span>;
+export interface TransactionErrorCounts {
+  [transactionId: string]: number;
+}
+export type TraceItems = Array<Transaction | Span>;
+export interface TraceAPIResponse {
+  trace: TraceItems;
```

Comment:
I don't think `TraceItems` should be a separate type. If we want to use `TraceItems` we should access `TraceAPIResponse[`trace`]`